common/pooll2tx.go

```diff
@@ -92,14 +92,17 @@ func NewPoolL2Tx(tx *PoolL2Tx) (*PoolL2Tx, error) {
 
 // SetType sets the type of the transaction
 func (tx *PoolL2Tx) SetType() error {
-	if tx.ToIdx >= IdxUserThreshold {
+	isAddrEmpty := tx.ToBJJ == EmptyBJJComp && tx.ToEthAddr == EmptyAddr
```

Comment:
I think the code should be more descriptive. It's not clear what it does at first glance. Let's avoid inline conditions in value attributions.

Reply:
Can you please give your suggestion? What changes were requested in this PR? can you be more clear?

Reply:
This statement has a inline condition to attribute a value.
```isAddrEmpty := tx.ToBJJ == EmptyBJJComp && tx.ToEthAddr == EmptyAddr```
I believe, for the sake of code readability, we would need to avoid this kind of statement.

Reply:
If you want an example, this is would be:
```var isAddrEmpty bool
if ToBJJ == EmptyBJJComp && tx.ToEthAddr == EmptyAddr {
    isAddrEmpty = true
} ``` 
Of course it does the same. But it is more clear to read because it's more descriptive.

Reply:
Why? is this a code style to follow? Is this your suggestion?

```go
if tx.ToBJJ == EmptyBJJComp && tx.ToEthAddr == EmptyAddr && tx.ToIdx >= IdxUserThreshold {
		tx.Type = TxTypeTransfer
	} else if tx.ToBJJ == EmptyBJJComp && tx.ToEthAddr == EmptyAddr && tx.ToIdx == 1 {
		tx.Type = TxTypeExit
	} else if tx.ToIdx == 0 {
		if tx.ToBJJ != EmptyBJJComp && tx.ToEthAddr == FFAddr {
			tx.Type = TxTypeTransferToBJJ
		} else if tx.ToEthAddr != FFAddr &&
			tx.ToEthAddr != EmptyAddr &&
			tx.ToBJJ == EmptyBJJComp {
			tx.Type = TxTypeTransferToEthAddr
		} else {
			return tracerr.Wrap(errors.New("malformed transaction"))
		}
	} else {
		return tracerr.Wrap(errors.New("malformed transaction"))
	}
	return nil
```